pkg/image/testdata/full-valid-example.yaml

```diff
@@ -5,6 +5,8 @@ image:
   baseImage: slemicro5.5.iso
   outputImageName: eibimage.iso
 operatingSystem:
+  installDevice: /dev/sda
+  unattended: true
```

Comment:
Can we update the [config parsing test](https://github.com/suse-edge/edge-image-builder/blob/main/pkg/image/definition_test.go#L11) here? It's what this definition file is used for.

Reply:
Done!